extensions/metadata/sleepycat/src/main/java/org/exist/storage/md/MetaDataImpl.java

```diff
@@ -99,14 +100,14 @@ public MetaDataImpl(Database db) throws IOException {
 
 		LOG.debug("ready ... ");
 
-		MetaDataImpl.inst = this;
-		MetaData.inst = this;
+		MetaDataImpl.instance = this;
```

Comment:
because 'org.exist.storage.md.MDStorageManager` don't know about `org.exist.storage.md.MetaDataImpl`. [look here](https://github.com/shabanovd/exist/blob/feature/metadata_storage/extensions/metadata/interface/src/main/java/org/exist/storage/md/MDStorageManager.java#L94)